src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1222,6 +1252,19 @@ def should_update_existing_container(self, c):
         # force_single_container_updates argument is set when cloud build is used to build the image and this argument ensures that only one container with the new cloud build image is present in the containerapp
         return c["name"].lower() == self.get_argument_container_name().lower() or self.get_argument_force_single_container_updates()
 
+    def set_up_runtime(self):
+        if self.get_argument_runtime() is None and self.get_argument_enable_java_metrics() is None:
```

Comment:
Here it can be:
if self.get_argument_runtime() is not None or self.get_argument_enable_java_metrics() is not None: